### PR TITLE
Upgrade Inline Forms

### PR DESCRIPTION
**Pull request by rhymiz**
This work is to make `inline_forms` Rails 5 compatible.

/CC: @acesuares

### CODE REVIEW

File: lib/app/views/inline_forms/_header.html.erb [hunk 1]
```diff
@@ -2,7 +2,7 @@
   <nav id='inline_forms_application_top_bar' class="top-bar" data-topbar>
     <ul class="title-area">
       <li class="name">
-        <h1><a href="/"><%= application_name %> v<%= inline_forms_version %></a></h1>
+        <h1><a href="/"><%= application_name %> v<%= inline_forms_version %> <%= current_user.name %></a></h1>
```
Comment (rhymiz) [inline lib/app/views/inline_forms/_header.html.erb]: Remember to remove this.

File: lib/app/controllers/inline_forms_controller.rb [hunk 2]
```diff
@@ -271,4 +284,7 @@ def unravel(deep_hash, level=-1)
     a
   end
 
+  def revert_params
+    params.require(:id).permit(:update, :comchannel)
```
Comment (acesuares) [inline lib/app/controllers/inline_forms_controller.rb]: Hmm it's weird to have comchannel here, don't remember why. It's not part of inline_forms (IF from now on) but from specifice implementations (i.e. business logic). Probably needed it to 'fix' something  :-1:

File: lib/app/views/inline_forms/_new.html.erb [hunk 3]
```diff
@@ -3,11 +3,9 @@
     <% flash.each do |key, value| %>
       <div class="flash <%= key %>">
         <ul>
-          <% value.each do |m| %>
             <li>
-              <%= m %>
+              <%= value %>
```
Comment (acesuares) [inline lib/app/views/inline_forms/_new.html.erb]: I turned out that value was an array sometimes ?

File: lib/app/controllers/inline_forms_controller.rb (hunk 2 above)
Comment (rhymiz) [inline lib/app/controllers/inline_forms_controller.rb]: Yeah, this was just put in place to not "break" certain things, still doing some cleanup work.

File: lib/app/views/inline_forms/_new.html.erb (hunk 3 above)
Comment (rhymiz) [inline lib/app/views/inline_forms/_new.html.erb]: This was "breaking" things - it's temporary.

File: .vscode/settings.json [hunk 4]
```diff
@@ -0,0 +1,3 @@
+// Place your settings in this file to overwrite default and user settings.
+{
+}
```
Comment (rhymiz) [inline .vscode/settings.json]: add this to `.gitignore`

File: lib/app/helpers/form_elements/pdf_link.rb [hunk 5]
```diff
@@ -17,17 +17,17 @@ def pdf_link_update(object, attribute)
...
-    else
-      distance_of_time_in_words_to_now_without_future(from_time, include_seconds) + ' ago'
-    end
-  end
-
-  alias_method_chain :distance_of_time_in_words_to_now, :future
-
-end
-
+#module ActionView::Helpers::DateHelper
```
Comment (rhymiz) [inline lib/app/helpers/form_elements/pdf_link.rb]: This breaks things... figure out why

File: lib/generators/templates/model_instance_human_attribute_name.rb [hunk 6]
```diff
@@ -0,0 +1,7 @@
+# Wrapper for @model.human_attribute_name -> Model.human_attribute_name
```
Comment (rhymiz) [inline lib/generators/templates/model_instance_human_attribute_name.rb]: This is unnecessary

File: lib/app/views/inline_forms/_new.html.erb [hunk 7]
```diff
@@ -27,16 +25,16 @@
     <% attributes.each do | attribute, name, form_element | -%>
       <% unless form_element.to_sym == :associated || form_element.to_sym == :tree || (cancan_enabled? && cannot?(:read, @Klass.to_s.underscore.pluralize.to_sym, attribute)) -%>
         <% css_class_id = "attribute_#{attribute}_#{@object.id}" -%>
-        <% if form_element == :header %>
+        <% if form_element && form_element.to_sym == :header %>
```
Comment (rhymiz) [inline lib/app/views/inline_forms/_new.html.erb]: messing with some things are breaking

**Pull request merged**